docs/components/content/examples/FormExampleElements.vue

```diff
@@ -80,7 +80,7 @@ async function submit (event: FormSubmitEvent<Schema>) {
     </UFormGroup>
 
     <UFormGroup name="checkbox" label="Checkbox">
-      <UCheckbox v-model="state.checkbox" />
+      <UCheckbox v-model="state.checkbox" label="Check me" />
```

Comment:
Why add a `label` here? This should with `UFormGroup` right?

Reply:
Checkbox components and Radio are a bit different from the others because a label is included in the component: 

https://github.com/nuxt/ui/pull/704/files#diff-eeb89ce166ecb860614d9cb79ad8b64476eaf9a919c31dfccb89c50052c2c728R21

Reply:
Yep, both are working! https://ui-git-fork-romhml-refactor-form-group-props-nuxt-js.vercel.app/dev/forms/form#input-events

Reply:
Does clicking on the `Checkbox` label works for you?

Reply:
Yeah sorry I could not see straight, just fixed it!

Reply:
The `label` of the `Radio` components doesn't seem to work while the one from the `FormGroup` does.

Reply:
Fixed, note that clicking on the FormGroup's label for Radio groups does not work properly, I'll fix this in another PR introducing a wrapper component (will also fix #684).